docs/examples/plotting/auto_color_scale.ipynb

```diff
@@ -450,15 +449,15 @@
     "                                     low_outlier=-3,\n",
     "                                     high_outlier=3,\n",
     "                                     background_noise=False)\n",
-    "ax, cb = plot_by_id(ds.run_id, auto_color_scale=True, cutoff_percentile=(0,5));"
+    "ax, cb = plot_dataset(ds, auto_color_scale=True, cutoff_percentile=(0,5));"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Defaults and customizing the auto color scaling<a name=\"customizing\"></a>\n",
-    "The defaults used by `plot_by_id` and `show_num` can be set in the `qcodesrc.json` files. The system default is described in `qcodes/config/qcodesrc.json` ([online](https://github.com/QCoDeS/Qcodes/blob/master/qcodes/config/qcodesrc.json))\n",
+    "The defaults used by `plot_dataset` can be set in the `qcodesrc.json` files. The system default is described in `qcodes/config/qcodesrc.json` ([online](https://github.com/QCoDeS/Qcodes/blob/master/qcodes/config/qcodesrc.json))\n",
```

Comment:
@jenshnielsen what should we do about these `qcodes/config/qcodesrc.json` kind of links?

Reply:
Just leave it for now i think